pkgs/applications/virtualization/gvisor/default.nix

```diff
@@ -0,0 +1,107 @@
+{ stdenv
+, buildBazelPackage
+, fetchFromGitHub
+, cacert
+
```

Comment:
can you remove that stray newline?

Reply:
Done. I will need to rebase again if #74536 is merged, but did this now.